### PR TITLE
RED-188638: Create template dashboards and alerts with DB tags

### PR DESCRIPTION
**Pull request by petar-valchev-redis**
Context: https://redislabs.atlassian.net/browse/RED-188638.

Adds a Grafana dashboard and example Prometheus alert built on the opt-in `db_tags` metrics.

Dashboard link [here](https://monitoring.qa.redislabs.com/grafana/d/afsvrhjarwq9sd/database-tags-dashboard-v2?orgId=1&from=now-6h&to=now&timezone=browser&var-cluster=aws-cluster-27236.env0.qa.redislabs.com&var-env=dev&var-env=prod&var-team=payments&var-team=platform&refresh=30s).

This is how the dashbaord looks for a cluster where db tags are enabled and `env` and `team` tags are exposed to metrics:
Screenshot 2026-07-22 at 16 39 16
Screenshot 2026-07-22 at 16 39 38

Steps for setup:
1. Create a cluster.
2. Enable DB tags: `rladmin metrics config expose_db_tags enabled`
3. Expose env and team: `rladmin metrics config metrics_tag_keys_exposed env,team`
4. Select the cluster, environment, and team in the Grafana dashboard.

---

> [!NOTE]
> **Low Risk**
> Changes are additive dashboard JSON, optional example alert rules, and documentation; no runtime application code or default alert wiring.
> 
> **Overview**
> Adds **observability templates** for opt-in `db_tags` Prometheus metrics so teams can slice Redis Enterprise database health by customer-defined labels (e.g. `env`, `team`).
> 
> A new Grafana v9–11 dashboard (`software/ops/database-tags.json`) joins cluster/env/team variables with `db_tags`, `db_config`, and standard DB metrics for tag export coverage, memory and ops by tag, top databases, and tagged vs untagged inventory. Docs in the dashboard catalog and Grafana README list the new asset.
> 
> An **example** Prometheus rule file fires when production-tagged databases exceed 90% memory for 5 minutes, with `promtool` unit tests covering firing, threshold edge cases, non-prod tags, and missing `db_tags`. Operators must enable tag export and customize label names/thresholds to match their taxonomy.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 1fc38a17699a738b7a82a08fccf8c323bd018b7a. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: grafana_v2/dashboards/grafana_v9-11/software/ops/database-tags.json [hunk 1]
```diff
@@ -0,0 +1,1018 @@
...
+        "definition": "label_values(db_tags, env)",
+        "hide": 0,
+        "includeAll": true,
+        "label": "Environment",
+        "multi": true,
+        "name": "env",
+        "options": [],
+        "query": {
+          "query": "label_values(db_tags, env)",
+          "refId": "StandardVariableQuery"
```
[the remark this answers is not in the document]
Comment (petar-valchev-redis) [reply grafana_v2/dashboards/grafana_v9-11/software/ops/database-tags.json:950]: fixed

**Review by slorello89 — APPROVED** [review]
LGTM 👍
